evap/results/tests/test_views.py

```diff
@@ -682,6 +683,28 @@ def test_textanswer_visibility_for_staff(self):
         self.assertNotIn(".other_responsible_orig_private.", page)
         self.assertNotIn(".other_responsible_orig_notreviewed.", page)
 
+    def test_textanswer_visibility_for_manager_own_export(self):
+        course = mommy.make(Course, state='published', can_publish_text_results=True)
+        questionnaire = mommy.make(Questionnaire)
+        question = mommy.make(Question, questionnaire=questionnaire, type="T")
+        course.general_contribution.questionnaires.set([questionnaire])
+        responsible = mommy.make(UserProfile)
+        responsible_contribution = mommy.make(Contribution, contributor=responsible, course=course, questionnaires=[questionnaire],
+            responsible=True, can_edit=True, comment_visibility=Contribution.ALL_COMMENTS)
+        manager_contribution = mommy.make(Contribution, contributor=self.manager, course=course, questionnaires=[questionnaire],
+           comment_visibility=Contribution.OWN_COMMENTS)
+
+        mommy.make(TextAnswer, question=question, contribution=course.general_contribution, answer="general_contribution", state=TextAnswer.PUBLISHED)
+        mommy.make(TextAnswer, question=question, contribution=responsible_contribution, answer="responsible_contribution", state=TextAnswer.PUBLISHED)
+        mommy.make(TextAnswer, question=question, contribution=manager_contribution, answer="manager_contribution", state=TextAnswer.PUBLISHED)
+        mommy.make(TextAnswer, question=question, contribution=manager_contribution, answer="manager_contribution_private", state=TextAnswer.PRIVATE)
+
+        page = self.app.get("/results/semester/{}/course/{}?view=export&contributor_id={}".format(course.semester.id, course.id, self.manager.id), user='manager')
+        self.assertNotIn("general_contribution", page)
+        self.assertNotIn("responsible_contribution", page)
+        self.assertIn("manager_contribution", page)
+        self.assertNotIn("manager_contribution_private", page)
```

Comment:
can you keep the structure of the other tests?

Reply:
or was there a reason for not doing that?

Reply:
we don't have that specific situation in the fixture data so i created everything we need here.

Reply:
would it work to add a responsible/contributor to the staff group, or is something missing then?

Reply:
that should be it. would you add an additional user or add the group to an existing one and remove it at the end of the test?
i thought you'd appreciate not using the fixture here as a start for a brighter future ;)

Reply:
yeah, but i didn't want to do that here because that seems like really a lot of stuff needed. so we could open an issue for replacing the fixture and here i'm changing the group memberships for this one test?